bears/python/PythonPackageInitBear.py

```diff
@@ -0,0 +1,46 @@
+import os
+import re
+
+from coalib.results.Result import Result
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.settings.ConfigurationGathering import (find_user_config)
+
+
+class PythonPackageInitBear(GlobalBear):
+    """
+    Looks for missing __init__.py files in directories containing python files.
+    """
+
+    LANGUAGES = {'Python', 'Python 3', 'Python 2'}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+
+    def run(self,
+            ignore_dirs: list = [],
+            create_empty_init_file: bool = True):
+        """
+        :param root_directory:
+            The root directory of the project whose tree has to be checked
+        :ignore_dirs:
+            List of directories to be ignored
+        """
+        files = sorted(self.file_dict)
+        dirs = set()
+        config_path = self.get_config_dir() or find_user_config(
+            os.getcwd()).strip('.coafile')
+        ignore_dirs = [os.path.join(config_path, dir) for dir in ignore_dirs]
```

Comment:
You're claiming the ignore mechanism of coala is broken. The truth is *you're using it wrong*. This makes it a usability issue and I can understand people wanting to use it this way. That can be fixed. But you deviating from everything we do by redoing the mechanism is not at all helpful.